README.md

```diff
@@ -25,14 +31,35 @@ To improve code quality, we run linters, type checkers, and unit tests. The
 tools can be run using [nox]. We recommend installing nox using [pipx] to have
 it available globally:
 
+### Install [nox]
```

Comment:
The heading get a bit too fine-grained. What about keeping it simple and create one big bash code fence:

### Nox

```bash
# install
python -m pip install pipx
python -m pipx install nox

# run all sessions
nox

# list all sessions
nox -l

# run individual sessions
nox -s session_name
```

Reply:
I added what you suggest. I would even suggest that the rest of the text goes into the doc string of the sessions to simplify the README

Reply:
I added back one paragraph, which I think is important.